yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -122,28 +122,58 @@ Internet-Draft                 YANG Semver                  October 2023
    submodule has non-backwards-compatible (NBC) changes compared to its
    previous revision, and a scheme that uses the revision history as a
```

Comment:
Is the text on "scheme that uses..." still valid here?

Reply:
Good question.  I struggled with this.  I ended up using "scheme" as a generic noun for a means to do something.  The module versioning still has considerable text about what "derived" means.  It's more simplified to be sure, but it's there.  I'm certainly open to suggestion on how this text in Semver might best reflect that module versioning allows for branching and Semver is describing that.